### ISSUE TITLE
USER ISSUE: Crashed while pulling my wood cart

### ISSUE THREAD

**Issue by flying7eleven** [StrangeLoopGames/EcoIssues#18361]
\
**Version:** 0.9.0.2 beta release-71

**Steps to Reproduce:**
1. Just pulling my cart up a ramp

**Expected behavior:**
Not crashing

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: InvalidOperationException
Message:Collection was modified; enumeration operation may not execute.
Source:System.Private.CoreLib

System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Generic.List`1.Enumerator.MoveNextRare()
   at Eco.Shared.Utils.EnumerableExtensions.ToBson(IEnumerable enumeration, INetClient client, Boolean doAllParams)
   at Eco.Shared.Serialization.BsonManipulator.ToBson(Object val, INetClient client, Boolean useReflection)
   at Eco.Core.Controller.ControllerManager.PackageValue(IMvcNetClient boundClient, Object value, Int32 nameID)
   at Eco.Core.Controller.ControllerManager.PackageMember(IMvcNetClient boundClient, IController controller, Int32 nameID)
   at Eco.Core.Controller.ControllerManager.GetChanges(IMvcNetClient client)
   at Eco.Plugins.Networking.Client.Update()
   at Eco.Plugins.Networking.Client.<.ctor>b__70_1()</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No, no mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@Kaerius @flying7eleven Can you provide some additional information?
1. Does this happen all the time? or one, a couple of times?
2. Please provide log file if it happens again. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
3. What server are you playing on?

**Comment by SlayksWood** [comment]
merged to #18214

**Comment by Kaerius** [comment]
The error is repeated regularly.
I attach the logs.
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/5262879/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5262880/Player.log)
I play on the server "[WIPE 19/09] UA/RU Средняя сложность ECO Сервер Рус Язык"
